### PR TITLE
Update TrustFrameworkKeySet Policy Keys Document for keys_v2

### PR DESCRIPTION
**Pull request by gyanendersinghgithub**
In TrustFrameworkPolicies, secrets are stored which allow the framework to access different services on behalf of the application. Introducing new functionality using keys_v2 to support entity type operations like get keys and patch the key status for a key within keyset.

### CODE REVIEW

File: api-reference/beta/api/trustframeworkkeyset-patchkeystatus.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+|Authorization|Bearer {token}. Required. Learn more about [authentication and authorization](/graph/auth/auth-concepts).|
+| Content-type  | application/json. Required. |
+
+## Request body
+
+In the request body, provide a JSON object with the following parameters.
+
+| Parameter    | Type        | Description |
+|:-------------|:------------|:------------|
+|status|String|Similar to the **status** property of **trustFrameworkKey_v2**.|
```
Comment (tblazek98) [inline api-reference/beta/api/trustframeworkkeyset-patchkeystatus.md:46]: Is status a string, not a boolean?

Comment (gyanendersinghgithub) [reply api-reference/beta/api/trustframeworkkeyset-patchkeystatus.md:46]: Internally it is an enum and for api users, it is string only.

Comment (tblazek98) [reply api-reference/beta/api/trustframeworkkeyset-patchkeystatus.md:46]: If it's enum, then let's list out the possible values here as well

Comment (gyanendersinghgithub) [reply api-reference/beta/api/trustframeworkkeyset-patchkeystatus.md:46]: It is mentioned in the api-reference/beta/resources/trustFrameworkkey_v2.md and description points to it.

**Comment by Lauragra** [comment]
@gyanendersinghgithub , FYI, this is the public repo for external contributions. For new API docs, please use the private authoring repo so that you can follow our workflow: https://github.com/microsoftgraph/microsoft-graph-docs/pulls.

For more information, see the Get started topic: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/614277/Get-started

**Pull request closed without merging**